Framework/API/src/FileFinder.cpp

```diff
@@ -424,9 +398,9 @@ FileFinderImpl::getArchiveSearch(const Kernel::FacilityInfo &facility) const {
   return archs;
 }
 
-std::string
-FileFinderImpl::findRun(const std::string &hintstr,
-                        const std::vector<std::string> &exts) const {
+std::string FileFinderImpl::findRun(const std::string &hintstr,
```

Comment:
Can we leave the old function override as well?

Reply:
Same as above except I'm not sure why I added a default value to exts. Old cold calls findRun with exts and hintstr, so the majority usecase will be with `useExtsOnly=false`, which does not change the behaviour from before